src/averageAges.js

```diff
@@ -14,12 +14,28 @@
  *
  * @return {number}
  */
+function calculatePersonsAverageAge(persons) {
+  const personAges = persons.map(person => person.died - person.born);
+  const averageAgePerson = (personAges.reduce((sum, age) =>
+    (sum + age)) / personAges.length);
+
+  return averageAgePerson;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // const century =
+  let men = people.filter(person => person.sex === 'm');
+
+  if (century) {
```

Comment:
Everything looks good just replace if with ?

Reply:
i'd say it's fine to keep it like this